.github/workflows/spelling.yml

```diff
@@ -2,14 +2,21 @@ name: Spell checking
 on:
   pull_request_target:
   push:
+  issue_comment:
 
 jobs:
   build:
     name: Spell checking
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2.0.0
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
       with:
-        ref: ${{ github.event.pull_request.head.sha }}
-        fetch-depth: 5
-    - uses: check-spelling/check-spelling@0.0.17-alpha
+        ref: refs/pull/${{github.event.pull_request.number}}/merge
+    - name: checkout
+      if: "!contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
+    - uses: check-spelling/check-spelling@v0.0.18
+      with:
+        experimental_apply_changes_via_bot: 1
```

Comment:
This enables talking to the bot. If it messes up, you can remove this section to disable it.

Reply:
Just found two 🪲s. This feature isn't ready for repositories that take non-member PRs/have other bots.